app/routes/_layout+/assets.$assetId.tsx

```diff
@@ -174,6 +205,38 @@ export async function action({ context, request, params }: ActionFunctionArgs) {
         return json(data({ success: true }));
       }
 
+      case "set-reminder": {
+        const payload = parseData(formData, setReminderSchema);
+        const hints = getHints(request);
+
+        const fmt = "yyyy-MM-dd'T'HH:mm";
+
+        const alertDateTime = DateTime.fromFormat(
+          formData.get("alertDateTime")!.toString()!,
+          fmt,
+          {
+            zone: hints.timeZone,
+          }
+        ).toJSDate();
+
+        await createAssetReminder({
+          ...payload,
+          assetId: id,
+          alertDateTime,
+          organizationId,
+          createdById: userId,
+        });
+
+        sendNotification({
+          title: "Reminder created",
+          message: "A reminder for you asset has been created successfully.",
+          icon: { name: "success", variant: "success" },
+          senderId: authSession.userId,
+        });
+
+        return json(data({ success: true }));
```

Comment:
I can see it returns json, but when i used the modal to create a new reminder, it redirected me to overview, even tho i was on alerts index. Because we create them from a modal that can be triggered on every tab on the page, we should always stay on the tab the user currently is, not redirect to overview.